src/NuGet.Core/NuGet.Commands/CommandRunners/PackCommandRunner.cs

```diff
@@ -131,10 +131,67 @@ public PackageArchiveReader BuildPackage(PackageBuilder builder, string outputPa
                 PrintVerbose(outputPath, builder);
             }
 
+            if (_packArgs.InstallPackageToOutputPath)
+            {
+                _packArgs.Logger.LogMinimal(String.Format(CultureInfo.CurrentCulture, Strings.Log_PackageCommandInstallPackageToOutputPath, "Package", outputPath));
+                WriteResolvedNuSpecToPackageOutputDirectory(outputPath, builder);
+                WriteSHA512PackageHash(outputPath);
+            }
+
             _packArgs.Logger.LogMinimal(String.Format(CultureInfo.CurrentCulture, Strings.Log_PackageCommandSuccess, outputPath));
```

Comment:
the output path will be the full path to the .nupkg file - you don't want to log that the nupkg was output when you are actually installing it in an extracted manner.

Reply:
Not sure I understand this feedback. This is actually our desired logging for the build tools, I want to know where the files are going to be output and if the package creation succeeded.

Here is an example of what the logging looks like now.
Install Package to output path 'd:\packages\microsoft.nuget.packagingexample.nupkg'
Install NuSpec to output path 'd:\packages\microsoft.nuget.packagingexample.nuspec'
Install SHA512 to output path 'd:\packages\microsoft.nuget.packagingexample.nupkg.sha512'
Successfully created package 'd:\packages\microsoft.nuget.packagingexample.nupkg'.